### PR TITLE
[MWPW-139963] Table style fixes

### PR DESCRIPTION
**Pull request by seanchoi-dev**
This PR will fix some of style issue with table block

- RTL icon margin-right to left to fix this issue (icon is too closed to the test on RTL pages)
image

,
- Removing flex-direction: column style to avoid the alignment issue for mobile:
image

,
- Extending the small mobile media to 430px to avoid fat button like this (it happens at 403px screen size):
image

,
- Filters are too closed:
image

,
Resolves: [MWPW-139963](https://jira.corp.adobe.com/browse/MWPW-139963)

**Test URLs:**
General table test page:
- Before: https://main--milo--adobecom.hlx.page/docs/library/blocks/table?martech=off
- After: https://table-mobile-style-fix--milo--adobecom.hlx.page/docs/library/blocks/table?martech=off

Page that is reported:
- Before: https://main--milo--adobecom.hlx.page/drafts/Soujanya/mediablockstory/tabelblocktest?martech=off
- After: https://table-mobile-style-fix--milo--adobecom.hlx.page/drafts/Soujanya/mediablockstory/tabelblocktest?martech=off

RTL Page that is reported:
- Before: https://main--cc--adobecom.hlx.page/ae_ar/drafts/Anupama/Table/table-cc?martech=off
- After: https://main--cc--adobecom.hlx.page/ae_ar/drafts/Anupama/Table/table-cc?milolibs=table-mobile-style-fix&martech=off

### CODE REVIEW

**Review by sharath-kannan — APPROVED** [review]

**Review by elan-tbx — APPROVED** [review]

File: libs/blocks/table/table.css [hunk 1]
```diff
@@ -590,14 +595,9 @@ header.global-navigation {
   }
 }
 
-@media (max-width: 400px) {
+@media (max-width: 430px) {
```
Comment (mokimo) [inline libs/blocks/table/table.css:598]: Not blocking the PR, but I'm not particularly happy with having non-standard breakpoints. @overmyheadandbody wdyt?

Comment (mokimo) [inline libs/blocks/table/table.css:598]: Not blocking the PR, but I'm not particularly happy with having non-standard breakpoints. @overmyheadandbody wdyt?

Comment (robert-bogos) [reply libs/blocks/table/table.css:598]: I also agree that we should be consistent with breakpoints 👍

**Comment by seanchoi-dev** [comment]
This is ready to merge.

**Pull request merged**